container/rules/aggregate_debian_packages.bzl

```diff
@@ -0,0 +1,116 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_docker//container:container.bzl", _container = "container")
+load("@base_images_docker//package_managers:download_pkgs.bzl", _download = "download")
+load("@base_images_docker//package_managers:install_pkgs.bzl", _install = "install")
+load("@base_images_docker//package_managers:apt_key.bzl", _key = "key")
+
+debian_pkgs_attrs = _container.image.attrs + _key.attrs + _download.attrs + {
+    # Redeclare following attributes as non-mandatory.
+    "image_tar": attr.label(
+        allow_files = True,
+        single_file = True,
+    ),
+    "image": attr.label(
+        allow_files = True,
+        single_file = True,
+    ),
+    "packages": attr.string_list(),
+    "keys": attr.label_list(
+        allow_files = True,
+    ),
+    "output_image_name": attr.string(),
+}
+
+def _aggregate_debian_packages_impl(ctx):
```

Comment:
Are we reusing logics in https://github.com/bazelbuild/bazel-toolchains/blob/master/container/rules/docker_toolchains.bzl. This rule should go into that file, or at least use part of the logic there.